variables.tf

```diff
@@ -4,6 +4,12 @@ variable "region" {
   description = "AWS region"
 }
 
+# Needed for https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest#one-nat-gateway-per-availability-zone
+variable "azs" {
```

Comment:
Where is this variable used exactly?

(ref. https://github.com/jenkins-infra/aws/pull/261/files#diff-49b6dfd157f2e0513e99f20fea69f13b7ef66d1007b5555b44994848543b7edaR9 :) )

Reply:
I don't see it in the docs, just the mention they must be declared:

<img width="1078" alt="image" src="https://user-images.githubusercontent.com/91831478/195845261-587b6b96-1922-4c78-b73e-d96f4e81b36e.png">

Reply:
In the context of this documentation (screenshot), "variables" means "child module's variables": https://www.terraform.io/language/modules/syntax#calling-a-child-module .

So `vars.azs` is the input attribute `azs = <...>` that I linked earlier, which is set to the value from a data source.

Reply:
> that I linked earlier

Ah OK
Your link returned nothing particular (no line selected nothing)

Reply:
>  Your link returned nothing particular (no line selected nothing)

My bad. Here is the link: https://github.com/jenkins-infra/aws/blob/main/vpc.tf#L9